snapcraft/cli.py

```diff
@@ -132,12 +132,13 @@
         ],
     ),
     craft_cli.CommandGroup(
-        "Store Validation Sets",
```

Comment:
we should probably keep this name and create a new group for registries.

Assertions is not a word we should expose